CHANGELOG.md

```diff
@@ -2,12 +2,18 @@
 
 ## [Unreleased](https://github.com/TykTechnologies/tyk-helm-chart/tree/HEAD)
 
-[Full Changelog](https://github.com/TykTechnologies/tyk-helm-chart/compare/v0.12.0...HEAD)
+## [v0.13.0](https://github.com/TykTechnologies/tyk-helm-chart/tree/HEAD)
```

Comment:
```suggestion
## [v0.13.0](https://github.com/TykTechnologies/tyk-helm-chart/tree/v0.13.0)
```

Reply:
Discussed on slack; no need for modifications.